### PR TITLE
Change the objects and arrays into classes

### PR DESCRIPTION
**Pull request by denisdiaconu**
In this project, we refactor the Awesome books application to use JavaScript classes.

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -1,52 +1,78 @@
-const title = document.getElementById('title');
-const author = document.getElementById('author');
-const btn = document.getElementById('btn');
-const storageList = document.getElementById('storage_list');
-if (!localStorage.getItem('count')) {
-  localStorage.setItem('count', 0);
-}
-if (!localStorage.getItem('books')) {
-  localStorage.setItem('books', JSON.stringify([]));
+class Ls {
```
Comment (smcommits) [inline src/js/index.js:1]: - [ ] **The newly added book disappears when the page is refreshed. Please fix this.**

When I add a book, the data is persisted in the local storage. Great job with that. However, the book disappears from the page when I refresh it. Please fix it. 

a

Comment (smcommits) [inline src/js/index.js:1]: - [ ] **[OPTIONAL] Please consider using more descriptive names in your code.**

The name `Ls` makes it difficult to understand the code. You can name the class something like `Books` for example.

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @denisdiaconu,  

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!


 ## Required Changes ♻️
_Please check the inline comments below._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by denisdiaconu** [comment]
The newly added book is now appearing when the page is refreshed.

**Review by de-scholar — APPROVED** [review]
Hi @denisdiaconu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by de-scholar — APPROVED** [review]
Hi @denisdiaconu and @TSHEPO-CLOUD , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**